src/Illuminate/Foundation/stubs/facade.stub

```diff
@@ -5,7 +5,7 @@ namespace DummyNamespace;
 use Illuminate\Support\Facades\Facade;
 
 /**
- * @see \DummyTarget
+ * @mixin \DummyTarget
```

Comment:
This is incorrect. The facade methods are static, not instance.

Reply:
I did not find a standard definition of mixin but it worked very well in the latest version of phpstorm in practice.